book/src/pallets/nft/extrinsics.md

```diff
@@ -1 +1,40 @@
 # Extrinsics
+
+## Create Class
```

Comment:
If I am not mistaken it would be considered generated? The struct NFTclass provided by composable-traits seems to just define a u8 integer that get's passed.